Meta-Tag creation crahses shortening long URLs

## Reproduction

Open the Windows-Executable of Release Version 0.1.0 on Windows 11 64bit. In the Menu select `Tools > Create Meta Tags...`. In the new Window at **Video (v) URL** enter

``` url
https://www.youtube.com/watch?v=oHmqgxXyn4Iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
```

results in an Error and no Meta-Tag displayed, see attated image:

![grafik](https://github.com/bohning/usdb_syncer/assets/8994014/e31f674b-6b62-4a58-964f-91e605728cbd)

The error Message in the GUI is

``` log
2023-08-12 13:15:07 [ERROR] Uncaught exception:
    Traceback (most recent call last):
      File "usdb_syncer\gui\meta_tags_dialog.py", line 166, in _update_output
      File "usdb_syncer\gui\meta_tags_dialog.py", line 187, in _try_shorten_url
      File "usdb_syncer\gui\meta_tags_dialog.py", line 211, in _try_shorten_url
      File "pyshorteners\__init__.py", line 32, in __init__
      File "importlib\__init__.py", line 126, in import_module
      File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1142, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'pyshorteners.shorteners'
```

Full console log since Application start in the terminal window is

``` log
2023-08-12 13:12:10 [DEBUG] Starting new HTTPS connection (1): usdb.animux.de:443
2023-08-12 13:12:10 [DEBUG] https://usdb.animux.de:443 "GET /?link=profil HTTP/1.1" 200 None
2023-08-12 13:12:10 [DEBUG] Loading KWallet
2023-08-12 13:12:10 [DEBUG] Loading SecretService
2023-08-12 13:12:10 [DEBUG] Loading Windows
2023-08-12 13:12:10 [DEBUG] Loading chainer
2023-08-12 13:12:10 [DEBUG] Loading libsecret
2023-08-12 13:12:10 [DEBUG] Loading macOS
2023-08-12 13:12:11 [DEBUG] https://usdb.animux.de:443 "POST / HTTP/1.1" 200 None
2023-08-12 13:12:11 [INFO] Successfully logged in to USDB with user 'apps_of_mh'.
2023-08-12 13:12:11 [DEBUG] Post request for https://usdb.animux.de/index.php
2023-08-12 13:12:11 [DEBUG] https://usdb.animux.de:443 "POST /index.php?link=list HTTP/1.1" 200 None
2023-08-12 13:12:11 [INFO] Fetched 0 new song(s) from USDB.
DirectWrite: CreateFontFaceFromHDC() failed (Gibt einen Fehler in einer Eingabedatei an, zum Beispiel eine Schriftartdatei.) for QFontDef(Family="8514oem", pointsize=9, pixelsize=20, styleHint=5, weight=400, stretch=100, hintingPreference=0) LOGFONT("8514oem", lfWidth=0, lfHeight=-20) dpi=96
2023-08-12 13:12:15 [INFO] Application successfully loaded.
2023-08-12 13:15:07 [ERROR] Uncaught exception:
    Traceback (most recent call last):
      File "usdb_syncer\gui\meta_tags_dialog.py", line 166, in _update_output
      File "usdb_syncer\gui\meta_tags_dialog.py", line 187, in _try_shorten_url
      File "usdb_syncer\gui\meta_tags_dialog.py", line 211, in _try_shorten_url
      File "pyshorteners\__init__.py", line 32, in __init__
      File "importlib\__init__.py", line 126, in import_module
      File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1142, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'pyshorteners.shorteners'
```

## Differentiation

The error occurs when the resulting metatags starting with `#VIDEO:v=` + the entered URL exceed 255 Characters. The URL

``` url
https://www.youtube.com/watch?v=oHmqgxXyn4Iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
```

results in exactly 255 / 255 Characters Meta-Tag String, which does not show any error.

--- a/src/usdb_syncer/gui/meta_tags_dialog.py
+++ b/src/usdb_syncer/gui/meta_tags_dialog.py
@@ -2,13 +2,13 @@
 
 from typing import Callable, Literal
 
-from pyshorteners.shorteners import tinyurl
 from pyshorteners.exceptions import (
     BadAPIResponseException,
     BadURLException,
     ExpandingErrorException,
     ShorteningErrorException,
 )
+from pyshorteners.shorteners import tinyurl
 from PySide6.QtGui import QGuiApplication
 from PySide6.QtWidgets import QDialog, QWidget
 

--- a/src/usdb_syncer/gui/meta_tags_dialog.py
+++ b/src/usdb_syncer/gui/meta_tags_dialog.py
@@ -8,6 +8,16 @@ from pyshorteners.exceptions import (
     ExpandingErrorException,
     ShorteningErrorException,
 )
+
+# Note: prevent import error in production by importing shorteners directoy instead of
+#
+#         from pyshorteners import Shortener
+#
+#       which loads all shorteners from all python modules from its subfolder. Creating
+#       a build with pyinstaller seems to leave the submodules out, because submodules
+#       can then not be imported in a production build with error
+#
+#         ModuleNotFoundError: No module named 'pyshorteners.shorteners'
 from pyshorteners.shorteners import tinyurl
 from PySide6.QtGui import QGuiApplication
 from PySide6.QtWidgets import QDialog, QWidget

--- a/src/usdb_syncer/gui/meta_tags_dialog.py
+++ b/src/usdb_syncer/gui/meta_tags_dialog.py
@@ -8,16 +8,6 @@ from pyshorteners.exceptions import (
     ExpandingErrorException,
     ShorteningErrorException,
 )
-
-# Note: prevent import error in production by importing shorteners directoy instead of
-#
-#         from pyshorteners import Shortener
-#
-#       which loads all shorteners from all python modules from its subfolder. Creating
-#       a build with pyinstaller seems to leave the submodules out, because submodules
-#       can then not be imported in a production build with error
-#
-#         ModuleNotFoundError: No module named 'pyshorteners.shorteners'
 from pyshorteners.shorteners import tinyurl
 from PySide6.QtGui import QGuiApplication
 from PySide6.QtWidgets import QDialog, QWidget

--- a/src/usdb_syncer/gui/meta_tags_dialog.py
+++ b/src/usdb_syncer/gui/meta_tags_dialog.py
@@ -2,7 +2,7 @@
 
 from typing import Callable, Literal
 
-from pyshorteners import Shortener
+from pyshorteners.shorteners import tinyurl
 from pyshorteners.exceptions import (
     BadAPIResponseException,
     BadURLException,
@@ -213,7 +213,7 @@ def _urls_and_setters(tags: MetaTags) -> list[tuple[str, Callable[[str], None]]]
 
 def _try_shorten_url(url: str) -> str:
     try:
-        short = Shortener().tinyurl.short(url).removeprefix("https://")
+        short = tinyurl.Shortener().short(url).removeprefix("https://")
     except (
         BadAPIResponseException,
         BadURLException,

